src/de/willuhn/jameica/fibu/ext/hibiscus/UmsatzListMenu.java

```diff
@@ -273,12 +274,53 @@ private Buchung createBuchung(Umsatz u, boolean auto) throws Exception
       Konto haben = template.getHabenKonto();
       if (auto && haben == null)
         throw new ApplicationException(i18n.tr("Buchungsvorlage \"{0}\" enth�lt kein Haben-Konto",template.getName()));
+      
+      //Bei Storno-Buchungen drehen wir soll und haben um
+      //Wenn beides vom typ Bankkonto ist (zb. bei Verbindlichkeitskonten) Konto mit name "Bank" als Bankkonto verwenden
+      if((u.getBetrag() > 0.1d && haben.getKontoArt().getKontoArt() == Kontoart.KONTOART_GELD && (soll.getKontoArt().getKontoArt() != Kontoart.KONTOART_GELD || haben.getName().equals("Bank")))
```

Comment:
Das mit dem hartcodierten "Bank" - also abhängig von der Bezeichnung des Kontos Entscheidungen zu treffen, finde ich nicht schön.

Reply:
ja, finde ich auch nicht so gut. Ich habe jedoch keine andere Möglichkeit gefunden. Bei den allermeisten Buchung kommt das jedoch gar nicht zum tragen. Nur wenn Soll und Haben Konto des Templates ein Geldkonto sind wird geschaut welches der beiden die Bezeichnung "Bank" hat und dann dieses ersetzt. Sollte man kein Konto mit der Bezeichnung "Bank" haben oder verwende wir das Konto in dem Fall nicht ersetzt, was auch nicht weiter schlimm ist, muss dann halt bei der manuellen Buchungs-Prüfung korrigiert werden. Un das Konto mit der Bezeichnung "Bank" ist ja in beiden Kontenrahmen enthalten, also quasi auch vorgegeben.